zebra/zserv.c

```diff
@@ -81,6 +81,8 @@ static int zsock;
 enum zserv_client_event {
 	/* Schedule a socket read */
 	ZSERV_CLIENT_READ,
+	/* Schedule a client socket read event, without polling an fd */
+	ZSERV_CLIENT_READ_EVENT,
```

Comment:
This a a `zserv_client_event.ZSERV_CLIENT_READ_EVENT`

The extra EVENT doesn't really add anything to help the developer understand its purpose looking only at this data structure. They would need to understand the difference between `thread_add_read` and `thread_add_event`

Maybe `ZEBRA_CLIENT_READ_CONTINUE` ?